pom.xml

```diff
@@ -272,6 +272,13 @@
                 </container>
               </configuration>
             </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <configuration>
+                <source>${java-version}</source>
+              </configuration>
+            </plugin>
```

Comment:
- [ ] It has been added to `profile travis`, is it correct?

Reply:
It was a mistake. Corrected the description location of `maven-javadoc-plugin`